rhba-dev-env/vagrant/README.md

```diff
@@ -0,0 +1,178 @@
+2) enp0s31f6
+3) docker0
+4) virbr0
+```
+
+ After that ansible playbook will start and all configuration will be performed automatically.
+ 
+ If all tasks are executed successfully you will see something like:
+ 
+```bash
+ PLAY RECAP *********************************************************************
+workstation                : ok=29   changed=3    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0   
+
+Friday 26 July 2019  01:59:18 +0000 (0:00:08.785)       0:12:35.106 *********** 
+=============================================================================== 
+setup_env : upgrade all packages -------------------------------------- 427.18s
+setup_env : install the latest version of docker ----------------------- 97.67s
+setup_env : install the latest version of cekit packages --------------- 75.63s
+setup_env : clone repo https://github.com/jboss-container-images/rhpam-7-openshift-image.git -- 53.78s
+setup_env : install useful packages ------------------------------------ 27.99s
+setup_env : clone repo https://github.com/jboss-container-images/rhdm-7-openshift-image.git -- 20.85s
+setup_env : install the latest version of ansible ---------------------- 11.04s
+setup_env : install useful packages for brew --------------------------- 10.41s
+setup_env : clone repo https://gitlab.cee.redhat.com/mmagnani/rhba-dev.git --- 8.79s
+setup_env : install redhat-internal-cert -------------------------------- 5.39s
+setup_env : start docker service ---------------------------------------- 4.67s
+Gathering Facts --------------------------------------------------------- 1.57s
+setup_env : docker storage setup ---------------------------------------- 1.34s
+setup_env : enable @cekit/cekit ----------------------------------------- 1.26s
+setup_env : enable docker service --------------------------------------- 0.84s
+setup_env : adding user vagrant to group docker ------------------------- 0.77s
+setup_env : docker storage configurations ------------------------------- 0.71s
+setup_env : ensure group "docker" exists -------------------------------- 0.66s
+setup_env : create kerberos configuration ------------------------------- 0.61s
+setup_env : copy rpm redhat-internal-cert ------------------------------- 0.55s
+```
+
+Inside the Virtual Machine
+--------------
+
+After everything is set up the first step is to restart the virtual machine to ensure that all configurations are consistent.
+
+```bash
+vagrant ssh
+sudo reboot
+```
+Log in again and go to the cloned repository at the virtual machine startup:
+
+```bash
+vagrant ssh
+cd /home/vagrant/workspace/kie-cloud-tools/rhba-dev-env/ansible
+```
+Run ssh-copy-id for localhost (use the password "vagrant")
+
+```bash
+ ssh-copy-id localhost
+/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/vagrant/.ssh/id_rsa.pub"
+The authenticity of host 'localhost (::1)' can't be established.
+ECDSA key fingerprint is SHA256:NGzRYlpK+S1jsn5CENT6V+obbav7vzAOLcOYrWcw3hs.
```

Comment:
Perhaps change this to "SHA256:XXXXXXXXXX" for the purpose of instructions.

Reply:
done!